src/jail/options.rs

```diff
@@ -686,6 +687,21 @@ impl JailOptions {
                     seccomp_profile_name = String::from("go");
                     execve_args.extend([format!("./{}", args.run_target)]);
                 }
+                args::Language::Julia => {
+                    extra_memory_size_in_bytes = JULIA_EXTRA_MEMORY_SIZE_IN_BYTES;
+                    seccomp_profile_name = String::from("jl");
```

Comment:
seems like the seccomp files are missing from this change! can they be added?

Reply:
Sure - I don't know what those are though 😅